### ISSUE TITLE
foot: config not working probably

### ISSUE THREAD

**Issue by KarukSophia** [NixOS/nixpkgs#107072]
**Describe the bug**
The foot config isn't loaded at `.config/foot/foot.ini`. This can be worked around by having it instead be `.config/footrc`. However when gets errors even with the standard foot template config:
```
[user@nixos:~/.config]$ foot
info: main.c:307: version: 1.4.4
info: main.c:314: arch: x86_64
info: config.c:1061: loading configuration from /home/user/.config/footrc
 err: config.c:834: /home/user/.config/footrc:21: invalid section name: scrollback
 ```
 The config used:
 https://codeberg.org/dnkl/foot/src/branch/master/foot.ini
 
**Notify maintainers**
@sternenseemann 

**Metadata**
```
[user@nixos:~/.config]$ nix-shell -p nix-info --run "nix-info -m"
 - system: `"x86_64-linux"`
 - host os: `Linux 5.4.81, NixOS, 20.09.2205.65c9cc79f1d (Nightingale)`
 - multi-user?: `yes`
 - sandbox: `yes`
 - version: `nix-env (Nix) 2.3.9`
 - channels(root): `"nixos-20.09.2205.65c9cc79f1d, unstable-21.03pre257780.e9158eca70a"`
 - channels(user): `""`
 - nixpkgs: `/nix/var/nix/profiles/per-user/root/channels/nixos`
```

**Comment by sternenseemann** [comment]
The issues you are experiencing are because of [changes](https://codeberg.org/dnkl/foot/releases/tag/1.5.0) between foot 1.4.x (which you are using) and foot 1.5.x (which is available on nixos-unstable).

More specifically:

* 1.5.0 introduced the new location for the config file `$XDG_CONFIG_HOME/foot/foot.ini`
* 1.5.0 added the new `scrollback` config section which replaced the old `scrollback`  setting name

I think you have two options:

* Update to `nixos-unstable` or pull only foot from unstable
* Adjust your config to foot 1.4.x: Remove the `scrollback` section and move `scrollback.lines` to `scrollback`

`foot` is changing relatively quickly, so the upstream documentation may differ substantially from the package in the stable version of NixOS. Try referring to the man pages shipped with the package.

**Comment by sternenseemann** [comment]
Try using the [footrc shipped with 1.4.4](https://codeberg.org/dnkl/foot/src/tag/1.4.4/footrc) as a base configuration.

**Comment by sternenseemann** [comment]
@KarukSophia could you resolve the issue on your side?
